demisto_sdk/commands/common/tests/constants_regex_test.py

```diff
@@ -28,6 +29,51 @@ def verify(acceptable, unacceptable, matched_regex):
         assert not checked_type(test_path, compared_regexes=matched_regex)
 
 
+def get_test_new_classifiers_paths(folder):
+    acceptable_code_files = {
```

Comment:
```suggestion
    acceptable_classifiers_paths = {
```

same change in all tests

Reply:
done